Formula/dovecot.rb

```diff
@@ -77,6 +77,15 @@ def install
   def caveats; <<-EOS.undent
     For Dovecot to work, you may need to create a dovecot user
     and group depending on your configuration file options.
+
+    Dovecot by default requires a higher limit on open files per
+    process than the Mac OS X standard of 256 open files. In order
+    to silence the warning from dovecot, the launchd plist sets
+    both hard and soft resource limits of 512 open files. It is
+    suggested you also set default_client_limit = 512 in
+    10-master.conf. If you are running a production server, set
```

Comment:
Ok. Going with option 4. Certainly more terse this way. Thank you for your feedback.